src/frontend/magnify/src/i18n/FormErrors/index.tsx

```diff
@@ -1,21 +1,17 @@
 import * as React from 'react';
-import { useEffect } from 'react';
 import { useIntl } from 'react-intl';
 import { setLocale } from 'yup';
 import { validationMessages } from '../Messages';
 
 export interface FormErrorsProps {}
 
-export function FormErrors(props: FormErrorsProps) {
+export function FormErrors() {
```

Comment:
Can you add a comment to explain the purpose of this component and how and where it should be used. I do not understand at first.

Reply:
Done